src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Task {
+
+    private String name;
+    private boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+    public void mark() {
+        this.isDone = true;
+    }
+
+    public void unmark() {
+        this.isDone = false;
```

Comment:
I think it'll be good to standardise where you want to use `this.isDone` or `isDone` in these methods. But this isn't really a violation since the use of `this` doesn't affect the functionality in these methods. 

Feel free to close this if you disagree